packages/curl/0.1.0/package.yml

```diff
@@ -0,0 +1,53 @@
+# curl package
+
+name: curl
+parent: default
+
+matches:
+  - trigger: ":curl:"
+    replace: "curl --request GET https://example.com"
+
+  - trigger: ":curlget:"
+    replace: "curl --request GET https://example.com?key=value"
+
+  - trigger: ":curlpost:"
+    replace: "curl --request POST https://example.com"
+
+  - trigger: ":curlput:"
+    replace: "curl --request PUT https://example.com"
+
+  - trigger: ":curldelete:"
+    replace: "curl --request DELETE https://example.com"
+
+  - trigger: ":curlpatch:"
+    replace: "curl --request PATCH https://example.com"
+
+  - trigger: ":curldata:"
+    replace: "curl --request POST --header 'Content-Type: multipart/form-data' --form 'key=value' http://example.com"
```

Comment:
Is there a reason for the mix of `https:` and `http:`? The former throughout might be safer? I'd be happy to merge if this is addressed (unless there are good reasons not to).

Reply:
Yes, I added the relevant global variables as per your prompt